zephyr/include/sof/trace/trace.h

```diff
@@ -7,38 +7,65 @@
 #ifndef __INCLUDE_TRACE__
 #define __INCLUDE_TRACE__
 
```

Comment:
@finikorg with this PR https://github.com/thesofproject/sof/pull/2882 I think we are just getting rid of trace classes.

Cc: @mmaka1

Reply:
yep, this is just to get thing going - we can update when functionality is working.